### ISSUE TITLE
Stylelint support ?

### ISSUE THREAD

**Issue by js2me** [sveltejs/eslint-plugin-svelte3#31]
Hello!  
This plugin is supporting some validation or linter for styles (.css) ?

Just currently most of time of development I need to fix similar code style
![image](https://user-images.githubusercontent.com/16340911/60628811-cb058700-9dfb-11e9-8c08-bb8b0c85d2b8.png)

**Comment by Conduitry** [comment]
This plugin will present Svelte's compiler errors and warnings - including those about the styles - but other than that, linting styles is well outside the scope of this plugin, and of ESLint in general. For something like Stylelint, I imagine you probably wouldn't even need a special Svelte plugin, and could simply use a plugin that extracts inline styles from HTML files.

**Comment by MaxMilton** [comment]
Stylelint already handles linting `*.svelte` files. Just include the file extentions you want to lint, e.g. in your `package.json#scripts`:

```json
"lint:css": "stylelint **/*.{css,html,js,md,mjs,svelte,svg,ts}",
```

Linting in VS Code (and other code editors too probably) also works if you set the language for `*.svelte` as `html`. In your project `.vscode/settings.json` try:

```json
"files.associations": {
    "*.svelte": "html"
  }
```

**Comment by mvolkmann** [comment]
@maxmilton Could you provided a little more detail on how you configured `stylelint-config-css-modules` to support use of `:global` in `.svelte` files?

**Comment by maxlath** [comment]
@mvolkmann AFAIU, you don't need `stylelint-config-css-modules`, just this rule:
```js
'selector-pseudo-class-no-unknown': [ true, { ignorePseudoClasses: 'global' } ]
```
